src/chunk.js

```diff
@@ -31,6 +31,9 @@ const TAG = 'CHUNK';
  */
 let deactivated = /nochunking=1/.test(self.location.hash);
 let allowLongTasks = false;
+const supportsInputPending =
+  'scheduling' in self.navigator &&
```

Comment:
Nit: can se just do `!!(self.navigator.scheduling && self.navigator.scheduling.isInputPending)`?

Reply:
Unwinding back to this implementation since Closure Compiler cannot interpret this correctly (its type definition for navigator is not sufficient).